### PR TITLE
MRG, DOC: epochs getitem clarification

### PR DESCRIPTION
**Pull request by drammock**
@sappelhoff does this accord with the behavior you're seeing?

### CODE REVIEW

File: mne/utils/mixin.py [hunk 1]
```diff
@@ -109,12 +109,18 @@ def __getitem__(self, item):
...
-               If conditions are separated by '/', selects every item
-               containing every list tag (e.g. ['audio', 'left'] selects
-               'audio/left' and 'audio/center/left', but not 'audio/right').
+               experimental conditions. The list of names is treated as an
+               inclusive-or condition; if *none* of the provided names match
+               any epoch labels, a ``KeyError`` will be raised.
+
+               If epoch labels are tags separated by '/', then providing
+               multiple tags as *separate list entries* (e.g.,
+               ``epochs[['audio', 'left']]``) will also act as an inclusive-or
```
Comment (agramfort) [inline mne/utils/mixin.py:118]: isn't that repeating what is just said above?

Comment (sappelhoff) [reply mne/utils/mixin.py:118]: I think @drammock means to say that `[["left", "audio"]]` would work for both kinds of epochs:

1. those that use the `/` style: `"audio/left"`
1. those that have just single keywords: `"audio"` for one epoch, `"left"` for another epoch

File: mne/utils/mixin.py [hunk 2]
```diff
@@ -109,12 +109,18 @@ def __getitem__(self, item):
...
+               filter. If multiple tags are provided as *a single list entry*
+               (e.g., ``epochs['name_1/name_2']``), this selects every item
+               containing every list tag. For example, ``epochs['audio/left']``
+               selects 'audio/left' and 'audio/center/left', but not
+               'audio/right'.
```
Comment (sappelhoff) [inline mne/utils/mixin.py:123]: I think I would refer to "logical and" here, like you refer to the "logical inclusive-or" above.

**Review by sappelhoff — COMMENTED** [review]
Thanks, LGTM overall.

Minor: Further up in the example, the events are referred to as "auditory", later then as "audio" --> would be good to make this consistent.

**Review by agramfort — APPROVED** [review]

**Comment by drammock** [comment]
@sappelhoff OK for you?  Here's the rendered doc https://23418-1301584-gh.circle-artifacts.com/0/dev/generated/mne.Epochs.html#mne.Epochs.__getitem__

(note that I pushed a fix for `versionadded` being indented too far, but that's not reflected in the rendered doc linked above)

**Review by sappelhoff — APPROVED** [review]
Thanks @drammock - lgtm!

**Pull request merged**